Error in ni_trajkovic.cpp

PCL trunk cannot be compiled and produce this error:

```
Building CXX object apps/CMakeFiles/pcl_ni_trajkovic.dir/src/ni_trajkovic.cpp.o
[ 90%] Built target pcl_openni_3d_concave_hull
Scanning dependencies of target pcl_openni_3d_convex_hull
[ 90%] Building CXX object apps/CMakeFiles/pcl_openni_3d_convex_hull.dir/src/openni_3d_convex_hull.cpp.o
Linking CXX executable ../bin/pcl_openni_voxel_grid
[ 90%] Built target pcl_openni_voxel_grid
Scanning dependencies of target pcl_openni_boundary_estimation
[ 90%] Building CXX object apps/CMakeFiles/pcl_openni_boundary_estimation.dir/src/openni_boundary_estimation.cpp.o
Linking CXX executable test_outofcore
In file included from /home/victor/pcl/keypoints/include/pcl/keypoints/trajkovic_2d.h:173:0,
                 from /home/victor/pcl/apps/src/ni_trajkovic.cpp:44:
/home/victor/pcl/keypoints/include/pcl/keypoints/impl/trajkovic_2d.hpp: In instantiation of 'void pcl::TrajkovicKeypoint2D<PointInT, PointOutT, IntensityT>::detectKeypoints(pcl::TrajkovicKeypoint2D<PointInT, PointOutT, IntensityT>::PointCloudOut&) [with PointInT = pcl::PointXYZRGBA; PointOutT = pcl::PointXYZI; IntensityT = pcl::common::IntensityFieldAccessor<pcl::PointXYZRGBA>; pcl::TrajkovicKeypoint2D<PointInT, PointOutT, IntensityT>::PointCloudOut = pcl::PointCloud<pcl::PointXYZI>]':
/home/victor/pcl/apps/src/ni_trajkovic.cpp:248:1:   required from here
/home/victor/pcl/keypoints/include/pcl/keypoints/impl/trajkovic_2d.hpp:222:9: error: '((pcl::TrajkovicKeypoint2D<pcl::PointXYZRGBA, pcl::PointXYZI>*)this)->pcl::TrajkovicKeypoint2D<pcl::PointXYZRGBA, pcl::PointXYZI>::<anonymous>.pcl::Keypoint<pcl::PointXYZRGBA, pcl::PointXYZI>::keypoints_indices_' is not a variable in clause 'shared'
 #pragma omp parallel for shared (output, keypoints_indices_) num_threads (threads_)
         ^
/home/victor/pcl/keypoints/include/pcl/keypoints/impl/trajkovic_2d.hpp:219:13: warning: unused variable 'occupency_map_size' [-Wunused-variable]
   const int occupency_map_size (indices.size ());
             ^
In file included from /home/victor/pcl/keypoints/include/pcl/keypoints/trajkovic_3d.h:215:0,
                 from /home/victor/pcl/apps/src/ni_trajkovic.cpp:45:
/home/victor/pcl/keypoints/include/pcl/keypoints/impl/trajkovic_3d.hpp: In instantiation of 'void pcl::TrajkovicKeypoint3D<PointInT, PointOutT, NormalT>::detectKeypoints(pcl::TrajkovicKeypoint3D<PointInT, PointOutT, NormalT>::PointCloudOut&) [with PointInT = pcl::PointXYZRGBA; PointOutT = pcl::PointXYZI; NormalT = pcl::Normal; pcl::TrajkovicKeypoint3D<PointInT, PointOutT, NormalT>::PointCloudOut = pcl::PointCloud<pcl::PointXYZI>]':
/home/victor/pcl/apps/src/ni_trajkovic.cpp:248:1:   required from here
/home/victor/pcl/keypoints/include/pcl/keypoints/impl/trajkovic_3d.hpp:236:9: error: '((pcl::TrajkovicKeypoint3D<pcl::PointXYZRGBA, pcl::PointXYZI>*)this)->pcl::TrajkovicKeypoint3D<pcl::PointXYZRGBA, pcl::PointXYZI>::<anonymous>.pcl::Keypoint<pcl::PointXYZRGBA, pcl::PointXYZI>::keypoints_indices_' is not a variable in clause 'shared'
 #pragma omp parallel for shared (output, keypoints_indices_) num_threads (threads_)
         ^
/home/victor/pcl/keypoints/include/pcl/keypoints/impl/trajkovic_3d.hpp:233:13: warning: unused variable 'occupency_map_size' [-Wunused-variable]
   const int occupency_map_size (indices.size ());
             ^
[ 90%] Built target test_outofcore
Scanning dependencies of target pcl_openni_ii_normal_estimation
[ 90%] Building CXX object apps/CMakeFiles/pcl_openni_ii_normal_estimation.dir/src/openni_ii_normal_estimation.cpp.o
make[2]: *** [apps/CMakeFiles/pcl_ni_trajkovic.dir/src/ni_trajkovic.cpp.o] Error 1
make[1]: *** [apps/CMakeFiles/pcl_ni_trajkovic.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....

```

See:
http://www.pcl-users.org/Error-compiling-3d-rec-framework-application-td4033536.html
http://www.pcl-users.org/last-commit-error-in-ni-trajkovic-cpp-td4033523.html

--- a/keypoints/include/pcl/keypoints/impl/trajkovic_2d.hpp
+++ b/keypoints/include/pcl/keypoints/impl/trajkovic_2d.hpp
@@ -219,7 +219,7 @@ pcl::TrajkovicKeypoint2D<PointInT, PointOutT, IntensityT>::detectKeypoints (Poin
   const int occupency_map_size (indices.size ());
 
 #ifdef _OPENMP
-#pragma omp parallel for shared (output, keypoints_indices_) num_threads (threads_)
+#pragma omp parallel for shared (output) num_threads (threads_)
 #endif
   for (int i = 0; i < indices.size (); ++i)
   {

--- a/keypoints/include/pcl/keypoints/impl/trajkovic_3d.hpp
+++ b/keypoints/include/pcl/keypoints/impl/trajkovic_3d.hpp
@@ -233,7 +233,7 @@ pcl::TrajkovicKeypoint3D<PointInT, PointOutT, NormalT>::detectKeypoints (PointCl
   const int occupency_map_size (indices.size ());
 
 #ifdef _OPENMP
-#pragma omp parallel for shared (output, keypoints_indices_) num_threads (threads_)
+#pragma omp parallel for shared (output) num_threads (threads_)
 #endif
   for (int i = 0; i < indices.size (); ++i)
   {

